Makefile

```diff
@@ -297,6 +297,9 @@ enable-machine:
 	@echo "------------------------------------------------------------------"
 	@echo "eval \"$(docker-machine env freshwater)\""
 
+sync-geonode:
+	@docker-compose run --rm -w /usr/src/geonode uwsgi paver sync
```

Comment:
Add GeoNode's paver sync command.
This will load basic GeoNode's fixtures, including default superadmin.

Reply:
I don't have any proposed solution for that yet for catalog version.
This one is just Makefile command for dev env.